openwisp_utils/admin_theme/context_processor.py

```diff
@@ -1,14 +1,24 @@
+import logging
+import sys
+
 from django.apps import registry
 from django.conf import settings
 from django.urls import reverse
 
+from ..admin_theme.menu import build_menu_groups
 from . import settings as app_settings
 
 
-def menu_items(request):
```

Comment:
This breaks backward compatibility. I tried running this branch on openwisp-controller and I got this exception:

```python
ImportError at /admin/
Module "openwisp_utils.admin_theme.context_processor" does not define a "menu_items" attribute/class
``` 

Please ensure there's an alias, eg: 

```python
# Kept for backward compatibility
# Todo: remove in version 0.9.0
menu_items = menu_groups
```